### PR TITLE
Add API documentation to beatmaps routes

### PR DESCRIPTION
**Pull request by ethamitc**
I've added API documentation to the following routes;
- `/api/v2/beatmaps/lookup`
- `/api/v2/beatmaps/{beatmap}`

I was going to do more but I wanted to ensure that I was doing them properly before I continued.

### CODE REVIEW

File: app/Http/Controllers/API/BeatmapsController.php [hunk 1]
```diff
@@ -7,20 +7,62 @@
...
+     *
+     * Attribute                            | Notes
+     * -------------------------------------|------
+     * beatmapset                           | See [Beatmapset](#beatmapset)
+     * failtimes                            | |
+     * max_combo                            | |
+     *
+     * @urlParam beatmap required The ID of the beatmap.
+     *
+     * @response "See Beatmap object section.
```
Comment (nanaya) [inline app/Http/Controllers/API/BeatmapsController.php:40]: missing closing quote

File: app/Http/Controllers/API/BeatmapsController.php [hunk 2]
```diff
@@ -7,20 +7,62 @@
...
+     * ---
+     *
+     * ### Response format
+     *
+     * Returns [Beatmap](#beatmap) object.
+     * Following attributes are included in the response object when applicable,
+     *
+     * Attribute                            | Notes
+     * -------------------------------------|------
+     * beatmapset                           | See [Beatmapset](#beatmapset)
```
Comment (nanaya) [inline app/Http/Controllers/API/BeatmapsController.php:34]: This should mention `ratings` is included. Link to Beatmapset is probably not needed as it's already linked in Beatmap(Compact) section.

File: app/Http/Controllers/API/BeatmapsController.php [hunk 3]
```diff
@@ -7,20 +7,62 @@
...
 
         return json_item($beatmap, 'Beatmap', ['beatmapset.ratings', 'failtimes', 'max_combo']);
     }
 
+    /**
+     * Lookup Beatmap
+     *
+     * Returns beatmap.
+     *
+     *  ---
```
Comment (nanaya) [inline app/Http/Controllers/API/BeatmapsController.php:54]: one too many spaces after `*`

**Review by nanaya — CHANGES_REQUESTED** [review]

**Review by nanaya — APPROVED** [review]

**Comment by ethamitc** [comment]
Before this PR is merged, is it okay if I go document some more undocumented routes?

**Comment by peppy** [comment]
Please just open a new pull request.

**Pull request merged**